keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/EventController.kt

```diff
@@ -34,8 +33,6 @@ class EventController(
     @RequestParam("limit") limit: Int?
   ): List<PersistentEvent> {
     log.debug("Getting state history for: $id")
-    val resource = repository.getResource(id)
-    val events = repository.resourceEventHistory(id, limit ?: DEFAULT_MAX_EVENTS)
-    return actuationPauser.addApplicationActuationEvents(events, resource)
+    return resourceService.getEnrichedEventHistory(id, limit ?: DEFAULT_MAX_EVENTS)
```

Comment:
Logic to manipulate the resource event history for pauses/resumes is now centralized in `ResourceService`.